### ISSUE TITLE
Clipping the URL of a page yields an empty "quote"

### ISSUE THREAD

**Issue by wsw70** [jplattel/obsidian-clipper#52]
Clipping selected text in the body o the page works fine.

Clipping the URL, however, yields an empty quote:

```
> 

Clipped from [Meilleure Serrure connectée août 2022 : quel modèle installer ?](https://www.phonandroid.com/meilleure-serrure-connectee-quel-modele-installer.html) at 2022-08-03.
```

**Comment by jplattel** [comment]
The current implementation work in a way that to user selects some text on a webpage to be clipped. How would you like it to work if no selection is made? I cannot clip the entire page, as it's too much data to pass along within the url (that's how it works technically)

Would you like the url to be the clipped text?

**Comment by wsw70** [comment]
> Would you like the url to be the clipped text?

Yes, this is what I was thinking. I tried to select the URL and have that posted to Obsidian.
OTOH I see that the clip includes the URL so maybe when just selecting the URL or not selecting anything, the default information (under the quote) could be copied?

**Comment by jimpfaff** [comment]
the reason you are having the problem is because of how you set up the "Clipping Format." You should check that area per the image below. You probably have it as:

> {clip}
Clipped from [{title}]({url}) at {date}.

What you need to change is to remove the beginning markdown item ">" out of that setting and then keep the "{clip}" there. Therefore the quote markdown will not transfer to your clip. And if you don't highlight anything on the page, no "{clip}" will be pushed up to your new note.

![image](https://user-images.githubusercontent.com/23179321/182672669-df57447b-d1f7-4115-b60b-7d626ef77984.png)
